velox/vector/tests/VectorMaker.h

```diff
@@ -193,6 +193,19 @@ class VectorMaker {
     return flatVector;
   }
 
+  /// Create a DECIMAL FlatVector from native DECIMAL values, precision, scale.
+  ///
+  /// Elements are nullable.
+  ///
+  /// Examples:
+  ///   auto flatVector = decimalFlatVectorNullable({ShortDecimal(1),
```

Comment:
It is a bit verbose to wrap each value in a ShortDecimal. Is there a way to allow for shorter syntax?

```
decimalFlatVectorNullable({1, 2, std::nullopt, 5}, DECIMAL(10, 1))
```

Reply:
yeah, I felt the same. Will improve.

Reply:
done.

Reply:
Looks so much nicer. Thank you!